app/views/spree/admin/shared/_routes.html.erb

```diff
@@ -1,10 +1,13 @@
 <script>
   Spree.routes = <%== {
-    :variants_search       => admin_search_variants_path(:format => 'json'),
+    :variants_search       => spree.api_variants_path(:format => 'json'),
     :taxons_search         => spree.api_taxons_path(:format => 'json'),
     :user_search           => spree.admin_search_users_path(:format => 'json'),
     :product_search        => spree.api_products_path(:format => 'json'),
     :option_type_search    => spree.api_option_types_path(:format => 'json'),
-    :states_search         => spree.api_states_path(:format => 'json')
+    :states_search         => spree.api_states_path(:format => 'json'),
+    :orders_api            => spree.api_orders_path(:format => 'json'),
+    :stock_locations_api   => spree.api_stock_locations_path(:format => 'json'),
```

Comment:
I dont know where this is used in v2 but in ofn v2 there will only be one stock location, so, maybe no need for this option.